x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -75,6 +76,9 @@ const ActionsComponent: React.FC<ActionProps> = ({
   const getTimeline = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
   const { startTransaction } = useStartTransaction();
 
+  const isEnterprisePlus = useLicense().isEnterprise();
+  // const isEnterprisePlus = true;
```

Comment:
If the `isEnterprisePlus` const variable is already defined on `line 79`,  shouldn't the `line 80` comment be removed?

Reply:
woops, i forgot to remove this line after testing

Reply:
No worries. It happens